design-proposals/virtiofs-support/virtiofs-feature-gates.md

```diff
@@ -0,0 +1,69 @@
+# Overview
+Virtiofs allows sharing Secrets, ConfigMaps, ServiceAccounts and DownwardAPI (a.k.a., ConfigVolumes) in a dynamic way,
+i.e., any change in those objects are reflected in the VirtualMachine (VM) without restarting the VM. Moreover, PVCs may
+also be shared using virtiofs allowing multiple readers and writers in the shared PVCs.
+
+## Motivation
+Currently, the usage of feature gates are mixed with the feature control. The feature gate `ExperimentalVirtiofsSupport`
+controls whether virtiofsd runs as rootful container inside the launcher pod or not. Moreover, there is no way to
+disable it completely and this feature is not in GA.
+
+In order to GA virtiofs, VMs should be able to live migrate while sharing data with virtiofs. The virtiofsd’s live
+migration support will distinguish between different use cases:  sharing ConfigMaps, Secrets, DownwardAPI and
```

Comment:
You right. I have left it in Section "KubeVirt Feature Gates", in which we explain what the `EnableVirtioFsConfigVolumes` FG to avoid defining a term with the term itself.